src/data/feed/coinbase.js

```diff
@@ -0,0 +1,75 @@
+(function(fc) {
+    'use strict';
+
+    // https://docs.exchange.coinbase.com/#market-data
+
+    fc.data.feed.coinbase = function() {
+
+        var product = 'BTC-USD',
+            start = null,
+            end = null,
+            granularity = null;
+
+        var coinbase = function(cb) {
+            var params = [];
+            if (start != null) {
+                params.push('start=' + start.toISOString());
+            }
+            if (end != null) {
+                params.push('end=' + end.toISOString());
+            }
+            if (granularity != null) {
+                params.push('granularity=' + granularity);
+            }
+            var url = 'https://api.exchange.coinbase.com/products/' + product + '/candles?' + params.join('&');
+            d3.json(url, function(error, data) {
+                if (error) {
+                    return cb(error);
```

Comment:
Habit I suppose, it's a Node-ism I've picked up ([random example](https://github.com/joyent/node/blob/master/lib/fs.js#L1465)). It saves on an else statement or separate return statement.